src/index.css

```diff
@@ -1,13 +1,31 @@
+@tailwind base;
```

Comment:
- [ ] Your desktop implementation is beautiful, however, the mobile version has some alignment and overflow issues:

![image](https://user-images.githubusercontent.com/57421823/206546165-d1a91024-748b-4e97-9988-9ecabd15b6d4.png)

![image](https://user-images.githubusercontent.com/57421823/206546217-2a644175-6c7c-48b0-9f50-c89cd3338c52.png)

![image](https://user-images.githubusercontent.com/57421823/206546402-bfc3bd8f-c964-41f3-9b92-fd5ace81ee6b.png)

The implementation idea is great, just try to align and center the items for it to work well design-wise. 👍